docs/using-the-jdbc-driver/UsingTheJdbcDriver.md

```diff
@@ -108,7 +108,6 @@ The AWS JDBC Driver has several built-in plugins that are available to use. Plea
 |------------------------------------------------------------------------------------------------|---------------------------|------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | [Failover Connection Plugin](./using-plugins/UsingTheFailoverPlugin.md)                        | `failover`                | Aurora                 | Enables the failover functionality supported by Amazon Aurora clusters. Prevents opening a wrong connection to an old writer node dues to stale DNS after failover event. This plugin is enabled by default.                                                                                                                                                                                                                                                                                                           |
 | [Host Monitoring Connection Plugin](./using-plugins/UsingTheHostMonitoringPlugin.md)           | `efm`                     | Aurora                 | Enables enhanced host connection failure monitoring, allowing faster failure detection rates. This plugin is enabled by default.                                                                                                                                                                                                                                                                                                                                                                                       |
-| Aurora Host List Connection Plugin                                                             | `auroraHostList`          | Aurora                 | Retrieves Amazon Aurora clusters information. <br><br>**:warning:Note:** this plugin does not need to be explicitly loaded if the failover connection plugin is loaded.                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
Does it make sense to mark this plugin as deprecated? We can remove this deprecated plugin in 3-6 months. That would be enough time for users to adjust their code base and app configurations.